lib/mina/rails.rb

```diff
@@ -161,8 +172,7 @@ def check_for_changes_script(options={})
         'Migrating database'
 
       queue check_for_changes_script \
-        :check => 'db/migrate/',
-        :at => ['db/migrate/'],
+        :at => [ *migration_path ],
```

Comment:
since the default setting is `migration_paths` (plural) shouldn't this be the same?